README.md

```diff
@@ -1 +1,31 @@
-## RPi Control Center
+# RPi Control Center
+
+A control center application for Raspberry Pi, designed to manage and monitor various Raspberry Pi functionalities through a user-friendly interface.
+
+<img src="screenshots/en/img.png" width="75%" title="rpicc Activity">
+
+## Features
+- **System Monitoring**: View system information such as CPU usage, memory usage, and disk space.
+- **Process Management**: Monitor and manage running processes.
+- **Network Monitoring**: View network statistics and active connections.
+- **GPIO Control**: Control the GPIO pins on the Raspberry Pi.
+
+## Requirements
+- Raspberry Pi with Raspbian OS
+- Python 3.x
+- Required Python libraries:
+  - `gi`
+  - `psutil`
+  - `RPi.GPIO`
+
+## Installation
+- Install the required libraries: See req.txt files for the required libraries.
+
+## Usage
+- Run the application using the following command:
+  ```
+  python3 activity.py
+  ```
```

Comment:
```suggestion
```
They will be run from the home screen of sugar, so I don't think that is necessary

Reply:
Ok so rather giving tut on how to run activity on sugar will be useful like you see example of other sugar activity readme.